app/src/main/java/com/github/se/gomeet/ui/mainscreens/create/CreateEvent.kt

```diff
@@ -58,29 +69,51 @@ fun CreateEvent(nav: NavigationActions, eventViewModel: EventViewModel, isPrivat
   val url = remember { mutableStateOf("") }
   val isPrivateEvent = remember { mutableStateOf(false) }
 
+  var imageUri by remember { mutableStateOf<android.net.Uri?>(null) }
+  var imageBitmap by remember { mutableStateOf<ImageBitmap?>(null) }
+  val imagePickerLauncher =
+      rememberLauncherForActivityResult(contract = ActivityResultContracts.GetContent()) {
+          uri: android.net.Uri? ->
+        imageUri = uri
+        uri?.let { uriNonNull ->
+          val inputStream: InputStream? =
+              try {
+                nav.navController.context.contentResolver.openInputStream(uriNonNull)
+              } catch (e: Exception) {
+                e.printStackTrace()
+                null
+              }
+          inputStream?.let {
+            val bitmap = BitmapFactory.decodeStream(it)
+            imageBitmap = bitmap.asImageBitmap()
+          }
+        }
+      }
+
```

Comment:
I think it would be better if we add this in a new function in the EventViewModel